appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/service/AppSecUtil.java

```diff
@@ -0,0 +1,26 @@
+/*-
+ * Copyright (C) 2023 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full license.
+ */
+
+package com.vaadin.appsec.v8.service;
+
+import com.vaadin.server.VaadinService;
+
+public class AppSecUtil {
```

Comment:
Is this public API? If so, let's add a JavaDoc or otherwise let's make it package private.

Reply:
There was only one method and only the init listeners use it. I moved it to an abstract super class.